interfaces/Portalicious/.node-version

```diff
@@ -1 +1 @@
-v20
+v22
```

Comment:
@aberonni This might not look like a big change... but I thought: Why wait?

v20 is already "not-in-line" with the other Portal and/or the other Services, so...

Reply:
I would wait simply because v22 is not stable yet, but tbh I'm ok either way because we're only using it for building really

Reply:
v22 _is_ stable already, right?

![image](https://github.com/user-attachments/assets/f18c0890-c270-4a6a-a4ef-0f2c16ed41e3)

https://endoflife.date/nodejs

Its not _the_ LTS version, because v20 isn't yet "out of LTS" so to speak..